### PR TITLE
Update storage-classes.md

### PR DESCRIPTION
**Pull request by Rajakavitha1**
Fixed the formatting issues that were incorrectly showing the variables as hyperlink texts that resulted in 404 errors

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/en/docs/concepts/storage/storage-classes.md [hunk 1]
```diff
@@ -627,25 +627,25 @@ parameters:
...
-* `fs`: filesystem to be laid out: [none/xfs/ext4] (default: `ext4`).
-* `block_size`: block size in Kbytes (default: `32`).
+* `fs`: filesystem to be laid out: `none/xfs/ext4` (default: <em>ext4</em>).
+* `block_size`: block size in Kbytes (default: <em>32</em>).
 * `repl`: number of synchronous replicas to be provided in the form of
-  replication factor [1..3] (default: `1`) A string is expected here i.e.
+  replication factor `1..3` (default: <em>1</em>) A string is expected here i.e.
   `"1"` and not `1`.
 * `io_priority`: determines whether the volume will be created from higher
-  performance or a lower priority storage [high/medium/low] (default: `low`).
```
Comment (msau42) [inline content/en/docs/concepts/storage/storage-classes.md]: The `</strong` doesn't look correct

**Comment by Hartmarken348** [comment]
11855

**Comment by Rajakavitha1** [comment]
@msau42 incorporated the changes you suggested.

**Comment by Hartmarken348** [comment]
11885

File: content/en/docs/concepts/storage/storage-classes.md (hunk 1 above)
Comment (Rajakavitha1) [reply content/en/docs/concepts/storage/storage-classes.md]: Thank you so much. Just removed </strong

Comment (Rajakavitha1) [reply content/en/docs/concepts/storage/storage-classes.md]: @msau42 Please take a look.

**Comment by msau42** [comment]
cc @kubernetes/sig-docs-en-owners 
On use of the `<em>` tag or if the markdown `*` should be used instead

**Comment by tengqm** [comment]
Don't get the point replacing "`" with "<em>"

**Comment by Rajakavitha1** [comment]
@tengqm incorporated the changes you suggested.
I had used <em> instead of ` because for some string example `"1"` it was difficult to identify the " vs `

Rajie

**Comment by tengqm** [comment]
/lgtm
/approve
Thanks.

**Pull request merged**